src/snowflake/cli/plugins/git/manager.py

```diff
@@ -3,16 +3,16 @@
 
 
 class GitManager(StageManager):
-    def show_branches(self, repo_name: str) -> SnowflakeCursor:
-        query = f"show git branches in {repo_name}"
+    def show_branches(self, repo_name: str, like: str) -> SnowflakeCursor:
+        query = f"show git branches like '{like}' in {repo_name}"
         return self._execute_query(query)
 
-    def show_tags(self, repo_name: str) -> SnowflakeCursor:
-        query = f"show git tags in {repo_name}"
+    def show_tags(self, repo_name: str, like: str) -> SnowflakeCursor:
+        query = f"show git tags like '{like}' in {repo_name}"
         return self._execute_query(query)
 
-    def show_files(self, repo_path: str) -> SnowflakeCursor:
-        query = f"ls {repo_path}"
+    def show_files(self, repo_path: str, pattern: str) -> SnowflakeCursor:
```

Comment:
How different is this from `StageManager.list_files`? Why not extend and use the parent method?

Reply:
Done with addition of
https://github.com/snowflakedb/snowflake-cli/pull/888